### DISCUSSION TITLE
记录一下 WSL2 更换内核的过程 🐱 ネコのメモ帳

### DISCUSSION THREAD

**Discussion by giscus** [MuelNova/blog#10] [Announcements]
# 记录一下 WSL2 更换内核的过程 🐱 ネコのメモ帳

最近准备开始看 Kernel Pwn 了，但是 wsl2 的内核似乎并不是原版的，所以进行一个替换。

https://n.ova.moe/blog/2023/02/13/_wsl2%e7%9a%84%e5%86%85%e6%a0%b8%e5%ae%89%e8%a3%85/

**Comment by Hepengzhou** [comment]
Kernel: arch/x86/boot/bzImage is ready  (#1)
***
*** The present kernel configuration has modules disabled.
*** To use the module feature, please run "make menuconfig" etc.
*** to enable CONFIG_MODULES.
***
make: *** [Makefile:1948: modules] Error 1 一路回车下的的 出这个是啥原因呢

**Reply by MuelNova** [reply]
这个 log 看不出来，能更详细一些么
